docs/content/latest/troubleshoot/cluster/recover_server.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Recover failed tserver/master
+linkTitle: Recover tserver/master
+description: Recover tserver/master
+aliases:
+  - /troubleshoot/cluster/recover-server/
+  - /latest/troubleshoot/cluster/recover-server/
+menu:
+  latest:
+    parent: troubleshoot-cluster
+    weight: 828
+isTocNested: true
+showAsideToc: true
+---
+
+You have a cluster running and now a tserver or master process/node has failed.
+Here are the steps to take to get the cluster back on shape:
+
+{{< note title="Note" >}}
+In this scenario we have a N-node setup, with replication factor (RF)=3.
+{{< /note >}}
+
+
+## Monitoring processes
+Its a good idea to have a cron/systemd setup that ensure that the yb-master/yb-tserver process is restarted if it is not running. 
+This handles transient failures (such as a node rebooting or process crash due to a bug/some unexpected behavior).
```

Comment:
Instead of starting this on a new line/para, merge in the same paragraph as previous sentence.